alpine/Dockerfile

```diff
@@ -1,19 +1,19 @@
-FROM alpine:latest
+FROM  alpine:latest
```

Comment:
Opinionated `dockerfmt` is opinionated. :laughing:

Reply:
a lot.